### DISCUSSION TITLE
Visual Studio docker image update

### DISCUSSION THREAD

**Discussion by xddxd** [cirruslabs/cirrus-ci-docs#921] [Q&A]
Hello, could the cirrusci/windowsservercore:visualstudio2019 image be updated so that it includes the latest version of Visual Studio?

**Comment by fkorotkov** [comment]
This change https://github.com/cirruslabs/docker-images-windows/commit/7cdd4dabba85a6b886c448f5a32946071ff97b4d should've invalidated the image.
